simphony/core/keywords.py

```diff
@@ -36,7 +36,7 @@
         name='ElectrostaticField',
         definition='Electrostatic field',  # noqa
         key='ELECTROSTATIC_FIELD',
-        shape=[],
+        shape=[1],
```

Comment:
@tuopuu we have to apply it to the metadata. Then I'll update this branch. First task for tomorrow.